### DISCUSSION TITLE
Matplotlib

### DISCUSSION THREAD

**Discussion by jerikeitzer** [CoBrALab/RABIES#346] [General]
Hello,
I am starting to set up RABIES and the installation went well but once i start I have this permissions issue with matplotlib

`Matplotlib created a temporary config/cache directory at /tmp/matplotlib-x_t4j376 because the default path (/home/rabies/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.`

I am using Docker on a macos 12.5.1

**Comment by jerikeitzer** [comment]
I have tried setting up a separate temp directory but it does not work. Any recommendations?

**Comment by Gab-D-G** [comment]
Hi, this is simply a warning message and can be ignored.

**Reply by gdevenyi** [reply]
We should try and silence this by handling this gracefully in the code:
https://stackoverflow.com/questions/9827377/setting-matplotlib-mplconfigdir-consider-setting-mplconfigdir-to-a-writable-dir

Or perhaps just setting the variable in the docker config to a writable path.
